### PR TITLE
[4.0] Group action buttons for com_fields

### PR DESCRIPTION
**Pull request by bembelimen**
### Summary of Changes
This PR continues the grouping of action buttons in the backend (see https://github.com/joomla/joomla-cms/pull/22832, https://github.com/joomla/joomla-cms/pull/22393, https://github.com/joomla/joomla-cms/pull/23337 and https://github.com/joomla/joomla-cms/pull/23336). This time the fields buttons.

### Testing Instructions
Go to any fields manager in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a field item.

### CODE REVIEW

File: administrator/components/com_fields/View/Fields/HtmlView.php [hunk 1]
```diff
@@ -158,53 +158,61 @@ protected function addToolbar()
...
+			}
+
+			if (Factory::getUser()->authorise('core.admin'))
+			{
+				$childBar->checkin('fields.checkin')->listCheck(true);
+			}
+
+			if ($canDo->get('core.edit.state'))
+			{
+				$childBar->trash('items.trash')->listCheck(true);
```
Comment (Hackwar) [inline administrator/components/com_fields/View/Fields/HtmlView.php]: Doesn't this have to be fields.trash instead of items.trash?

Comment (bembelimen) [reply administrator/components/com_fields/View/Fields/HtmlView.php]: Thanks, is fixed

**Comment by Hackwar** [comment]
I have tested this item :white_check_mark: successfully on 7ddb789c3a46186c2cb63b9bddfe73cf04d2e998<br /><br />works now as expected<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23370">issues.joomla.org/tracker/joomla-cms/23370</a>.</sub>

**Comment by PhocaCz** [comment]
Tested this in  4.0.0-alpha6-dev successfully :ok: :heavy_check_mark: 

OK

**Comment by drmenzelit** [comment]
I have tested this item :white_check_mark: successfully on 7ddb789c3a46186c2cb63b9bddfe73cf04d2e998<br /><br />Tested succesfully on Joomla! 4.0.0-alpha7-dev<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23370">issues.joomla.org/tracker/joomla-cms/23370</a>.</sub>

**Comment by PhocaCz** [comment]
I have tested this item :white_check_mark: successfully on 7ddb789c3a46186c2cb63b9bddfe73cf04d2e998<br /><br />Tested now in Joomla! Issue Tracker - CMS

Tested succesfully on Joomla! 4.0.0-alpha7-dev<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23370">issues.joomla.org/tracker/joomla-cms/23370</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23370">issues.joomla.org/tracker/joomla-cms/23370</a>.</sub>

**Pull request merged**